src/test/App.test.tsx

```diff
@@ -271,7 +291,7 @@ describe("App", () => {
     expect(await page.$eval(selectors.musicView, (e) => e.getAttribute("src"))).toEqual(
       "https://raw.githubusercontent.com/Church-Life-Apps/Resources/master/resources/images/shl/SHL_006.png"
     );
-  });
+  }, 40000);
```

Comment:
Does this need to be this? Maybe revert

Reply:
I would like to give it some time since some parts of it depends on internet speed (fetching image) and it's a pretty long test, I cut the timeout in half.